### ISSUE TITLE
High validation dice at the onset of training

### ISSUE THREAD

**Issue by ntajbakhsh** [ChengBinJin/V-GAN-tensorflow#11]
Nice work! Just have a question about the tensorboard results. From the figures posted under section Tensorboard, I understand that the dice for the validation set at iteration zero is about 0.79. I would assume a randomly initialized model would yield a lower level of performance. Are you by any chance training the model from a pre-trained network? Could you please comment on this?

**Comment by ChengBinJin** [comment]
I trained many different models and also trained from pre-trained weights that I trained with different hyper-parameters. When putting images of the Tensorboard section, it seems that these images are from pre-trained weight. If we initialized our weights with random initialization, the performance should be worse at first.
